src/main/kotlin/no/nav/familie/ba/sak/bisys/BisysController.kt

```diff
@@ -10,21 +10,16 @@ import java.time.YearMonth
 @RestController
 @RequestMapping("/api/bisys")
 @ProtectedWithClaims(issuer = "azuread")
-class BisysController {
+class BisysController(private val bisysService: BisysService) {
     @PostMapping(path = ["/hent-utvidet-barnetrygd"],
         consumes = [MediaType.APPLICATION_JSON_VALUE],
         produces = [MediaType.APPLICATION_JSON_VALUE])
     fun hentUtvidetBarnetrygd(@RequestBody request: BisysUtvidetBarnetrygdRequest): ResponseEntity<BisysUtvidetBarnetrygdResponse> {
-        return ResponseEntity.ok(BisysUtvidetBarnetrygdResponse(listOf(
-            UtvidetBarnetrygdPeriode(
-                BisysStønadstype.UTVIDET,
-                YearMonth.of(2020, 1),
-                YearMonth.of(2020, 1),
-                1024))))
+        return ResponseEntity.ok(bisysService.hentUtvidetBarnetrygd(request.ident, request.fraDato))
     }
 }
 
 class BisysUtvidetBarnetrygdRequest(val ident: String, val fraDato: LocalDate)
```

Comment:
```suggestion
class BisysUtvidetBarnetrygdRequest(val personIdent: String, val fraDato: LocalDate)
```

Kan det være nyttig å fortsette å bruke personIdent hos oss her også?

Reply:
Oi, det burde isåfall vært en del av API-diskusjonen.

Viktig kjepphest for oss nok til å forfølge, eller la det være?